Documentation/helm-values.rst

```diff
@@ -1381,6 +1381,10 @@
      - The priority class to use for cilium-agent.
      - string
      - ``""``
+   * - procfs
+     - 
+     - string
+     - ``"/host/proc"``
```

Comment:
Has this file been regenerated? I think you removed this option, and it shouldn't need to be exposed to the user any more...?